roles/openshift_logging/library/openshift_logging_facts.py

```diff
@@ -261,7 +261,7 @@ def facts_for_secrets(self, namespace):
     def facts_for_sccs(self):
         ''' Gathers facts for SCCs used with logging '''
         self.default_keys_for("sccs")
-        scc = self.oc_command("get", "scc", name="privileged")
+        scc = self.oc_command("get", "securitycontextconstraints", name="privileged")
```

Comment:
can we make this `securitycontextconstraints.v1.security.openshift.io` instead?

Reply:
Updated the PR accordingly (and rebased).